pjsip/src/pjsua-lib/pjsua_call.c

```diff
@@ -1475,6 +1516,8 @@ pj_bool_t pjsua_call_on_incoming(pjsip_rx_data *rdata)
     pjmedia_sdp_session *offer=NULL;
     pj_bool_t should_dec_dlg = PJ_FALSE;
     pjsip_tpselector tp_sel;
+    pj_str_t st_reason = pj_str("");
+    int st_code = 200;
```

Comment:
Why not initialize `st_code` with zero?

Reply:
It's following the original code on https://github.com/pjsip/pjproject/blob/05d03ad9ee4eb4d93f63d2eb7b4345e1ad49e615/pjsip/src/pjsua-lib/pjsua_call.c#L1561

Reply:
> It's following the original code on
> 
> https://github.com/pjsip/pjproject/blob/05d03ad9ee4eb4d93f63d2eb7b4345e1ad49e615/pjsip/src/pjsua-lib/pjsua_call.c#L1561

Oh, "the original code" is actually the default answer code for replaced dialog (may be overriden by app from the replaced dlg callback). IMO better not share the same var with the replaced dialog, it serves different purpose, it may cause confusion in the future.